### PR TITLE
REAL clock

### PR DESCRIPTION
**Pull request by mattprecious**

### CODE REVIEW

File: src/main/java/com/jakewharton/u2020/data/Clock.java [hunk 1]
```diff
@@ -3,4 +3,14 @@
 public interface Clock {
   long millis();
   long nanos();
+
+  public static final Clock REAL = new Clock() {
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/data/Clock.java]: `public static final` all redudant

**Comment by JakeWharton** [comment]
lgtm

File: src/main/java/com/jakewharton/u2020/data/Clock.java (hunk 1 above)
Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/data/Clock.java]: Ugh... terrible...

**Comment by JakeWharton** [comment]
missing Kochiku's "merge on success" here...

**Pull request merged**